test/core/test_lwt_list.ml

```diff
@@ -585,7 +580,15 @@ let suite = suite "lwt_list" [
       Lwt_list.partition_s (fun x -> f x >>= fun _ -> Lwt.return true) l in
     test_serialization m
   end;
+]
 
+let test_big_list m =
+  let make_list n = Array.to_list @@ Array.init n (fun x -> x) in
+  let f _ = Lwt.return () in
+  m f (make_list 10_000_000) >>= (fun _ -> Lwt.return_true)
+
+let suite_intensive = suite "lwt_list big lists"
+    ~only_if:(fun () -> Sys.getenv_opt "LWT_STRESS_TEST" <> None) [
```

Comment:
`getenv_opt` requires 4.05, so this won't compile on most versions of OCaml Lwt still supports.